### PR TITLE
Add note to explain authentication mesh policy CRD kind.

### PR DESCRIPTION
**Pull request by diemtvu**
Issue: https://github.com/istio/istio.io/issues/2950

### CODE REVIEW

**Comment by geeknoid** [comment]
To fix the CI failure, replace the use of > with {{< tip >}}

File: content/docs/tasks/security/authn-policy/index.md [hunk 1]
```diff
@@ -105,6 +105,9 @@ spec:
 EOF
 {{< /text >}}
 
+> The mesh authentication policy uses the same [API]((/docs/reference/config/istio.authentication.v1alpha1/)
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authn-policy/index.md]:
```suggestion
> The mesh authentication policy uses the [regular authentication policy API](/docs/reference/config/istio.authentication.v1alpha1/)
```

File: content/docs/tasks/security/authn-policy/index.md [hunk 2]
```diff
@@ -105,6 +105,9 @@ spec:
...
+ as regular authentication policy, except it is defined in the cluster-scoped CRD, `MeshPolicy`.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authn-policy/index.md]:
```suggestion
  it is defined in the cluster-scoped `MeshPolicy` CRD.
```

File: content/docs/tasks/security/authn-policy/index.md (hunk 1 above)
Comment (diemtvu) [reply content/docs/tasks/security/authn-policy/index.md]: Done

File: content/docs/tasks/security/authn-policy/index.md (hunk 2 above)
Comment (diemtvu) [reply content/docs/tasks/security/authn-policy/index.md]: Done

**Pull request merged**